src/main/java/io/github/programminglife2016/pl1_2016/parser/PositionHandler.java

```diff
@@ -0,0 +1,62 @@
+package io.github.programminglife2016.pl1_2016.parser;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by ravishivam on 23-4-16.
+ */
+public class PositionHandler {
```

Comment:
Please introduce an interface `PositionHandler` that contains a method `void calculatePositions()`. That helps DIP.